### ISSUE TITLE
Bash completion

### ISSUE THREAD

**Issue by lexischubert** [pp4rs/2021-uzh-installation-guide#2]
<img width="907" alt="Screen Shot 2021-08-12 at 6 18 20 PM" src="https://user-images.githubusercontent.com/60316653/129231867-3bee381f-bd03-48bf-a304-375ce18b66c0.png">

it goes on and on until it gives up: /usr/local/etc/bash_completion:1633: command not found: #

**Comment by lachlandeer** [comment]
thanks! 

new macs use the zsh shell now I think .... this is likely where the issue lies

**Comment by lachlandeer** [comment]
this is my attempted fix with the switch to zsh  

```
brew install zsh-completion
```

Then add the following to the `.zshrc`:

```
    if type brew &>/dev/null; then
        PATH=$(brew --prefix)/share/zsh-completions:$FPATH
        autoload -Uz compinit
        compinit
    fi
```

then source the zshrc


implemented in 8d081e6eecbd6b256093048578201ba916cbd475

open a new issue if this didn't do the trick
